src/diffusers/models/unet_1d.py

```diff
@@ -83,9 +85,10 @@ def __init__(
         mid_block_type: Tuple[str] = "UNetMidBlock1D",
         out_block_type: str = None,
         block_out_channels: Tuple[int] = (32, 32, 64),
+        layers_per_block: int = 2,
```

Comment:
This is a breaking change -> note that our configs already have set this to 1

Reply:
No, the indexing in the blocks files is changed to make the layer numbers more intuitive. Before `layers_per_block = 1` was 1 additional block, now it is reflected by the total number.